### DISCUSSION TITLE
Can GitHub mobile have widgets ?

### DISCUSSION THREAD

**Discussion by akhilsahuji** [community/community#77931] [Mobile] [answered]
GitHub mobile is amazing.
It can be more amazing if it had widget for Git heatmap for the home screen.

(maybe It will be a help for procrastinators like me.😅)

**Comment by techvishnu** [comment]
Agreed! A widget for the Git heatmap on the home screen would be awesome. It could help boost productivity and keep procrastination at bay. Looking forward to seeing this feature added!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @akhilsahuji ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Reply by rizwan123912** [reply]
love you

**Comment by iskae** [comment, marked as answer]
Hi @akhilsahuji 👋 

Thanks for reaching out ❤️ 

We have some widgets for the iOS app and planning for the Android App too 🎉 Let me create an internal tracking issue for the heatmap 🙇

Cheers!

**Reply by akhilsahuji** [reply]
Thanks  🙌

**Reply by Shmayash10** [reply]
.

**Reply by Waxo** [reply]
Can this widgets be available in iPhone standby mode also ? Would be a great addition

**Reply by Appfanatic456** [reply]
Do you know/ have a date for the android widgets?
